src/components/RadioButton/RadioButton.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { TPAComponentsWrapper } from '../../test/utils';
+import { radioButtonDriverFactory } from './RadioButton.driver';
+import { RadioButton } from './';
+import { radioButtonTestkitFactory } from '../../testkit';
+import { radioButtonTestkitFactory as enzymeRadioButtonTestkitFactory } from '../../testkit/enzyme';
+
+describe('RadioButton', () => {
+  const createDriver = createUniDriverFactory(radioButtonDriverFactory);
+
+  const defProps = {
+    onChange: () => {},
+    label: 'label',
+    value: 'value',
+  };
+
+  it('should render', async () => {
+    const driver = createDriver(<RadioButton {...defProps} />);
+    expect(await driver.exists()).toBe(true);
+  });
+
+  it('should show checked state', async () => {
+    const driver = createDriver(<RadioButton checked {...defProps} />);
+
+    expect(await driver.hasChecked()).toBeTruthy();
+  });
+
+  it('should show disabled state', async () => {
+    const driver = createDriver(<RadioButton disabled {...defProps} />);
+
+    expect(await driver.hasDisabled()).toBeTruthy();
+  });
+
+  describe('testkit', () => {
+    it('should exist', async () => {
+      expect(
+        await isUniTestkitExists(
+          <RadioButton {...defProps} />,
+          radioButtonTestkitFactory,
+          {
+            dataHookPropName: 'data-hook',
+          },
+        ),
+      ).toBe(true);
+    });
+  });
+
+  // describe('enzyme testkit', () => {
+  //   it('should exist', async () => {
+  //     expect(
+  //       await isUniEnzymeTestkitExists(
+  //         <RadioButton {...defProps}/>,
+  //         enzymeRadioButtonTestkitFactory,
+  //         mount,
+  //         {
+  //           dataHookPropName: 'data-hook',
+  //         },
+  //       ),
+  //     ).toBe(true);
+  //   });
+  // });
```

Comment:
Why is this commented out?

Reply:
im not sure what is this test checking

Reply:
So when the project is built, each component has testkits generated automatically.
These testkits expose the drivers we create, to our consumers, so that they can use them in their tests.
This test just checks that the testkit exists.